### PR TITLE
Add usage of checksum_dir and EXAMPLE_DIR doc.

### PR DESCRIPTION
**Pull request by us**
Related issue: #704
@Conchylicultor @pierrot0

### CODE REVIEW

File: docs/add_dataset.md [hunk 1]
```diff
@@ -607,6 +630,9 @@ class MyDatasetTest(tfds_test.DatasetBuilderTestCase):
       "name1": "path/to/file1",  # Relative to fake_examples/my_dataset dir.
       "name2": "file2",
   }
+  # If you want to change fake_data path:
```
Comment (pierrot0) [inline docs/add_dataset.md]: I would remove that part here.
That point is already touched in the other section.

File: docs/add_dataset.md [hunk 2]
```diff
@@ -564,7 +565,29 @@ The release note will be published for the next release.
...
+If you do not want to add a dataset to TFDS but want to use TFDS api, you can adjust some
+static paths according to your needs.
 
+### 1. Adjust checksums
+TFDS stores the checksum of downloaded files in `tensorflow_datasets/url_checksums`
+directory. `tensorflow_datasets.scripts.download_and_prepare` script adds to checksum
+files when run with `--register_checksums` flag.
+
+You can specify an alternative checksums directory like that:
+`tfds.add_checksum_dir = your/path`.
```
Comment (pierrot0) [inline docs/add_dataset.md]: Where is this defined?

**Comment by Conchylicultor** [comment]
Thank you for the doc. I've fixed minor typo and sent for internal review so it should be merged soon.

File: docs/add_dataset.md (hunk 2 above)
Comment (us) [reply docs/add_dataset.md]: before the testing.

**Comment by us** [comment]
@Conchylicultor @pierrot0 @cyfra Kokoro is still working and does not finish the build. 
I think Kokoro has a bug.

**Comment by Conchylicultor** [comment]
Kokoro had a bug but the PR was correctly merged: https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md#define-the-dataset-outside-tfds
Thanks for this. Closing now.

**Pull request closed without merging**